webapp/src/components/AssessmentPage.tsx

```diff
@@ -0,0 +1,479 @@
+import {
+  Box,
+  Chip,
+  Container,
+  Stack,
+  Button,
+  Paper,
+  TableRow,
+  TableHead,
+  TableContainer,
+  TableCell,
+  TableBody,
+  Table,
+  Tabs,
+  Tab,
+} from '@mui/material';
+import React, { useState } from 'react';
+import Badge, { BadgeProps } from '@mui/material/Badge';
+import { styled } from '@mui/material/styles';
+import CheckOutlinedIcon from '@mui/icons-material/CheckOutlined';
+import DoneAllOutlinedIcon from '@mui/icons-material/DoneAllOutlined';
+import ScheduleOutlinedIcon from '@mui/icons-material/ScheduleOutlined';
+import CancelOutlinedIcon from '@mui/icons-material/CancelOutlined';
+import ArchiveOutlinedIcon from '@mui/icons-material/ArchiveOutlined';
+import UpcomingOutlinedIcon from '@mui/icons-material/UpcomingOutlined';
+import LockClockOutlinedIcon from '@mui/icons-material/LockClockOutlined';
+import AssessmentIcon from '@mui/icons-material/Assessment';
+import { formatDateTime } from '../helpers/dateTime';
+interface AssessmentRow {
+  id: number;
+  title: string;
+  type: string;
+  dueDate: string;
+  testDuration: number;
+  submittedDate: string;
+  score: number;
+  availableDate: string;
+  status: string;
+}
+
+const assessmentList: AssessmentRow[] = [
```

Comment:
All looks good to me. the only advice may be to put the dummy data in somewhere else in another file and try to retrieve the data. It's just a little challenge and looks neat when peoples view it.

Reply:
Thank you @kaiyolau  for your feedback I did put the dummy data in separate folder